lib/matplotlib/ticker.py

```diff
@@ -1238,40 +1261,41 @@ def format_eng(self, num):
         u'-1.00 \N{GREEK SMALL LETTER MU}'
 
         `num` may be a numeric value or a string that can be converted
-        to a numeric value with the `decimal.Decimal` constructor.
+        to a numeric value with ``float(num)``.
```

Comment:
String were accepted before so I let it here. But other formatter classes do not seems to support such a feature, so if one thinks it is not worth supporting it anymore, it could be deprecated indeed.